### DISCUSSION TITLE
How to configure?

### DISCUSSION THREAD

**Discussion by ghost** [thevoltagesource/LennoxiComfort#27] [Q&A] [answered]
Hi! I apologize as I understand this is a fairly basic questions but I'm new to HA and this is the first custom integration I've used. I've installed this integration through HACS. I believe I understand the configuration variables/ values, but where do I actually change them? Is there an option in the HA / HACS interface or do I need to navigate to a particular configuration file to change directly?

The reason I ask is I do not see any associated entities to edit. I assume I may see one once properly configured but I'm not sure how to initially configure if I can't see an entitiy.

Thanks very much!

**Comment by thevoltagesource** [comment, marked as answer]
Hi @KieferMacK.  Unfortunately I don't have a config flow built to be able to configure this thru the UI. Configuration is completely manual. Once you install the integration thru HACS you will need to open the configuration.yaml file in your HA config folder and add the config entry with settings appropriate for your account. At minimum you need to add:

```
climate:
  - platform: myicomfort
    name: whatyouwantitcalledinha
    username: youricomfortusername
    password: youricomfortpassword
```

Once the yaml is saved HA just needs a reboot to make it live.  There are other options as listed in the readme of this repository but the above will get it going for you.

**Reply by ghost** [reply]
Thanks for the direction! This worked and I can how see the entity.

Having some trouble with the temperature showing as -2,000 C and commands not actually impacting the thermostat. I'll look into this more and may need to open another question. Again, thanks so much more creating and maintaining this even though you no longer use this particular thermostat!

**Reply by ghost** [reply]
Actually, I was able to solve the issue noted above by changing the system parameter from "0" to "1". I only have one system so I'm not sure what I was accessing with "0" but it seems to work great now! (Zone was kept as "0").

**Reply by thevoltagesource** [reply]
Cool! Kind of weird that Lennox marked it as system 1 instead of 0 but good job on figuring that out. Glad it is working for you.
